package.json

```diff
@@ -45,7 +49,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,css,json,md,ts,tsx}": [
+    "*.{js,css,scss,json,md,ts,tsx,html,yml}": [
       "prettier --write",
```

Comment:
```suggestion
      "npm run fix",
```

Reply:
This is better, because we don't want to run the whole thing for everything..